src/cases/containers/case-viewer-container/case-viewer-container.component.spec.ts

```diff
@@ -445,7 +445,7 @@ describe('CaseViewerContainerComponent - Hearings tab hidden', () => {
 
   it('should not display the Hearings tab', () => {
     component.appendedTabs$.subscribe((tabs) =>
-      expect(tabs.length).toEqual(0)
+      expect(tabs.length).not.toEqual(0)
```

Comment:
The test no longer tests the thing it is supposed to test. We need to understand why